examples/render-opengl/src/opengl.rs

```diff
@@ -40,12 +41,18 @@ pub fn launch_opengl_window() -> Result<App, Box<dyn Error>> {
 		.with_title("Render Inochi2D Puppet (OpenGL)");
 
 	let (window, gl_config) = glutin_winit::DisplayBuilder::new()
-		.with_preference(ApiPrefence::FallbackEgl)
+		.with_preference(ApiPreference::FallbackEgl)
 		.with_window_builder(Some(window_builder))
 		.build(&events, <_>::default(), |configs| {
 			configs
-				.filter(|c| c.srgb_capable())
-				.max_by_key(|c| c.num_samples())
+				.filter(|c| match c {
+					Config::Egl(c) => c.srgb_capable(),
+					_ => false,
```

Comment:
This should be `true` in the default case, otherwise it'll filter everything out and get the following error:
```
thread 'main' panicked at examples/render-opengl/src/opengl.rs:56:18:
called `Option::unwrap()` on a `None` value
```